A note in reference manual on the implicit status of main argument of t.(f) notation (#15460)
coq/coq: doc/sphinx/language/core/records.rst

> In each case, :token:`term0` is the object projected and the

Reviewer: ```suggestion
In each case, :token:`term0` is the projected object and the
```

other arguments are the parameters of the inductive type.

> Note that since the object projected is a component of the notation,
> it is always considered to be an explicit argument of :token:`qualid`,
> even if formally declared as implicit (see :ref:`ImplicitArguments`),

Reviewer: ```suggestion
Since the projected object is part of the notation,
it's always considered an explicit argument of :token:`qualid`,
even if it's formally declared as implicit (see :ref:`ImplicitArguments`),
```

If that still captures your meaning.

Author: Thanks!
